target/linux/bcm27xx/bcm2709/config-6.6

```diff
@@ -187,7 +187,6 @@ CONFIG_FB_CFB_FILLRECT=y
 CONFIG_FB_CFB_IMAGEBLIT=y
 CONFIG_FB_CORE=y
 CONFIG_FB_DEVICE=y
-CONFIG_FB_IOMEM_FOPS=y
 CONFIG_FB_IOMEM_HELPERS=y
```

Comment:
`CONFIG_FB_IOMEM_HELPERS` selects `CONFIG_FB_IOMEM_FOPS`, see https://github.com/torvalds/linux/blob/master/drivers/video/fbdev/core/Kconfig#L171
If the target config selects `CONFIG_FB_IOMEM_HELPERS=y` it should also select `CONFIG_FB_IOMEM_FOPS=y`.

Reply:
I understand that I should leave CONFIG_FB_IOMEM_FOPS=y for bcm27xx?

Reply:
> Done